extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -148,12 +159,17 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
       titleAndCloseWrapperEl.appendChild(titleEl);
     }
 
-    if (this.doesContainFormElement_()) {
+    const forms = this.element.querySelectorAll('form');
+    if (forms.length > 0) {
+      forms.forEach((form) => this.addMissingFormStatusElements_(form));
       // Page attachments that contain forms must display the page's publisher
       // domain above the attachment's contents. This enables users to gauge
       // the trustworthiness of publishers before sending data to them.
       this.headerEl.append(this.createDomainLabelElement_());
       this.headerEl.classList.add('i-amphtml-story-page-attachment-with-form');
+      this.storeService.dispatch(Action.ADD_TO_ACTIONS_ALLOWLIST, [
+        {tagOrTarget: 'FORM', method: 'submit'},
```

Comment:
What other actions do we need to support? Maybe `clear`?
https://amp.dev/documentation/components/amp-form/#actions

Reply:
Updated to support `clear`. I believe these two should be the only ones we need to support